shared/src/models/utils/radiogram-status/radiogram-status.ts

```diff
@@ -0,0 +1,8 @@
+import type { RadiogramAcceptedStatus } from './radiogram-accepted-status';
+import type { RadiogramDoneStatus } from './radiogram-done-status';
+import type { RadiogramUnreadStatus } from './radiogram-unread-status';
+
+export type RadiogramStatus =
```

Comment:
To be consistent with Activities, Events and Behaviors, this should be `ExerciseRatiogramStatus`.

Also, registering a new status will need changes in multiple files. Have a look at `exercise-simulation-event`, `exercise-simulation-activity` and so on.